From 691cb614f6bbe88e2d3aabaf1cf1c73e5e413173 Mon Sep 17 00:00:00 2001
From: Ladislav Marko <LadislavMarko@seznam.cz>
Date: Tue, 11 Apr 2023 15:51:06 +0200
Subject: [PATCH 110/381] doc: Fix incorrect pairing of functions

CLA: trivial

The functions that should be implemented together are `OSSL_FUNC_signature_verify_recover_init` and `OSSL_FUNC_signature_verify_recover` and not  `OSSL_FUNC_signature_verify_recover_init` with ` OSSL_FUNC_signature_verify_init`

Reviewed-by: Tomas Mraz <tomas@openssl.org>
Reviewed-by: Hugo Landau <hlandau@openssl.org>
(Merged from https://github.com/openssl/openssl/pull/20704)

(cherry picked from commit 51b941ac290864103d00a3d6a3018372b58b01f4)
---
 doc/man7/provider-signature.pod | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/doc/man7/provider-signature.pod b/doc/man7/provider-signature.pod
index 5a373c4c6e..4bffc9b7a3 100644
--- a/doc/man7/provider-signature.pod
+++ b/doc/man7/provider-signature.pod
@@ -155,7 +155,7 @@ set of "signature" functions, i.e. at least one of:
 
 =item OSSL_FUNC_signature_verify_init and OSSL_FUNC_signature_verify
 
-=item OSSL_FUNC_signature_verify_recover_init and OSSL_FUNC_signature_verify_init
+=item OSSL_FUNC_signature_verify_recover_init and OSSL_FUNC_signature_verify_recover
 
 =item OSSL_FUNC_signature_digest_sign_init, OSSL_FUNC_signature_digest_sign_update and OSSL_FUNC_signature_digest_sign_final
 
-- 
2.34.1

